pkg/pipelines/bootstrap.go

```diff
@@ -1,10 +1,63 @@
 package pipelines
 
-import "log"
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/mitchellh/go-homedir"
+	"sigs.k8s.io/yaml"
+)
 
 // Bootstrap is the main driver for getting OpenShift pipelines for GitOps
 // configured with a basic configuration.
 func Bootstrap(quayUsername, baseRepo, prefix string) error {
-	log.Printf("Bootstrapping %s, %s, %#v", quayUsername, baseRepo, prefix)
+	outputs := make([]interface{}, 0)
+
+	tokenPath, err := pathToDownloadedFile("token")
+	if err != nil {
+		return fmt.Errorf("failed to generate path to file: %w", err)
+	}
+	f, err := os.Open(tokenPath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	githubAuth, err := CreateOpaqueSecret("github-auth", f)
+	if err != nil {
+		return err
+	}
+	outputs = append(outputs, githubAuth)
+
+	authJsonPath, err := pathToDownloadedFile(quayUsername + "-auth.json")
```

Comment:
Our initial goal is to replicate the bootstrap script, but, yes, the idea is that it could be overridden by command-line options.

My thinking is that we'd pass these into the function from the CLI driver in `pkg/odo/pipelines`